doc/source/methods/ProtoSelect.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<ol><li>Since the method is technically not an explainer, I would rename the object variable to something else (same in example notebook).</li><li>At the top link to the API docs for a full description of all parameters?</li><li>[PLACHEHOLDER - to discuss last paragraph]</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<ol><li>Need to discuss what would be a proper naming</li><li>All the parameters are already presented</li><li>...</li></ol>